src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.csproj

```diff
@@ -9,6 +9,10 @@
     <NuspecFile Condition="'$(DotNetBuildFromSource)' == 'true'">$(MSBuildProjectName).netcoreapp.nuspec</NuspecFile>
     <IsPackable>true</IsPackable>
     <IsShipping>true</IsShipping>
+    <IsAspNetCoreApp>true</IsAspNetCoreApp>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageTags>files;filesystem</PackageTags>
+    <NoWarn>$(NoWarn);NU5100</NoWarn>
```

Comment:
I see a warning
```
error NU5100: The assembly 'tasks\netstandard2.0\Microsoft.Extensions.FileProviders.Embedded.Manifest.Task.dll' is not inside the 'lib' folder and hence it won't be added as a reference when the package is installed into a project. Move it into the 'lib' folder if it needs to be referenced. [C:\gh\aspnetcore2\src\FileProviders\Embedded\src\Microsoft.Extensions.FileProviders.Embedded.csproj]
```
This seems to only occur in aspnetcore builds and not in extensions. Perhaps we have a different sdk?